pandas_gbq/gbq.py

```diff
@@ -603,106 +458,56 @@ def delete_and_recreate_table(self, dataset_id, table_id, table_schema):
         table.create(table_id, table_schema)
         sleep(delay)
 
-
-def _parse_data(schema, rows):
```

Comment:
In test_gbq.py, there are several tests that have `gbq._parse_data(...)`. Could you update `test_gbq.py` as well? See the tests with prefix `test_should_return_bigquery*(...)`.